api-reference/sdk-reference/package-aqueduct/README.md

```diff
@@ -6,18 +6,15 @@
 * [`flow`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.flow)
 * [`operators`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.operators)
 * [`schedule`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.schedule)
-
```

Comment:
@vsreekanti I think these newlines are auto-removed by the doc generation script..

Reply:
Got it -- let's modify the doc generation script then.